src/hooks/tasks/useTasksByDay.ts

```diff
@@ -0,0 +1,70 @@
+import { TABLES } from 'services/supabase';
+import { ENTITY } from 'types';
+import { useQuery, useSelect } from '../supabase-swr/';
+
+export interface Meta {
+    uuid: string;
+}
+
+export interface Out {
```

Comment:
The name of this interface is odd, especially if it appears outside of this file which _sort of_ gives it context. I imagine _out_ is being used as a shorthand for _data out_ which is an alternative phrasing for _throughput_.